docs/stagenet.md

```diff
@@ -1,8 +1,9 @@
 # Joining the Stagenet/Sepolia network
 
-Currently, an initial version of the swap is deployed onto the Sepolia (Ethereum testnet) and Monero Stagenet networks. To join the network and try out the swap, either as a maker or a taker, please see the following.
+Swaps can be performed on the Sepolia (Ethereum testnet) together with the
+Monero Stagenet network. This document describes how to do stagenet swaps.
 
-> Note: a swap on stagenet currently takes around 10-20 minutes due to block times.
+> Note: stagenet and mainnet swaps currently takes around 20 minutes due to the 2-minute block time and required 10 confirmations before funds can be spent.
```

Comment:
I think you wanted to change the "takes" here

Reply:
also why mention mainnet here?

Reply:
I'll change this one to `take`